specification/communication/data-plane/CallAutomation/preview/2024-11-15-preview/communicationservicescallautomation.json

```diff
@@ -2230,6 +2230,107 @@
+          "description": "Container for details relating to the entity responsible for the creation of these call details.",
+          "$ref": "#/definitions/SourceDetails"
+        },
+        "sessionId": {
+          "description": "Id to exclusively identify this call session. IVR will use this for their telemetry/reporting.",
+          "type": "string"
+        },
+        "intent": {
+          "description": "The intent of the call",
+          "type": "string"
+        },
+        "callTopic": {
+          "description": "A very short description (max 48 chars) of the reason for the call. To be displayed in Teams CallNotification. ",
+          "type": "string"
+        },
+        "callContext": {
+          "description": "A summary of the call thus far. It will be displayed on a side panel in the Teams UI.",
+          "type": "string"
+        },
+        "transcriptUrl": {
+          "description": "Url for fetching the transcript of the call.",
+          "type": "string"
+        },
+        "callSentiment": {
+          "description": "Sentiment of the call thus far.",
+          "type": "string"
+        },
+        "suggestedActions ": {
+          "description": "Recommendations for resolving the issue based on the customer’s intent and interaction history.",
+          "type": "string"
+        }
+      }
+    },
+    "CallerDetails": {
+      "description": "Container for details relating to the original caller of the call.",
+      "type": "object",
+      "properties": {
+        "caller": {
+          "description": "Caller’s ID (MRI – can be phone number), Caller’s Display Name",
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "recordId": {
+          "description": "Caller’s record ID (ex in CRM). ",
+          "type": "string"
+        },
+        "screenPopUrl": {
+          "description": "Caller’s screen pop url. ",
+          "type": "string"
+        },
+        "isAuthenticated": {
+          "description": "Bit indicating whether the caller was authenticated.",
+          "type": "boolean"
+        },
+        "additionalCallerInformation ": {
+          "description": "A set of key value pairs (max 10, any additional entries would be ignored) which a bot author wants to pass to the Teams Client for display to the agent. ",
+          "type": "dictionary",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SourceDetails": {
+      "description": "Container for details relating to the entity responsible for the creation of these call details.",
+      "type": "object",
+      "properties": {
+        "source": {
+          "description": "ID of the source entity passing along the call details (MRI ex. Application Instance ID of - CQ/AA).",
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "language": {
+          "description": "Language of the source entity passing along the call details.",
```

Comment:
We should specify which standard is used for the language / locale code and give an example